### PR TITLE
feat: Extract common/lib/sandbox-packages

### PR DESCRIPTION
**Pull request by awais786**
Extract common/lib/sandbox-packages into separate repo.

https://2u-internal.atlassian.net/browse/BOM-2585
https://openedx.atlassian.net/browse/BOM-2585

Test release.
https://test.pypi.org/project/aq-codejail-includes/1.0.0/

Sandbox with loncapa problem

`loncapa` problem
https://studio-awais786.sandbox.edx.org/container/block-v1:edX+DemoX+Demo_Course+type@vertical+block@fa5f40832d7f4df9aee554ff26724c5b

Need some help about following problem.
```
    add a deprecated drag-and-drop problem type to Studio with custom grading code.
    add a custom problem type to Studio with custom grading code.
    include code that depends on eia and loncapa 
    publish and test both problems as a learner in LMS.
    run a problem report instructor task.
    check that the grader code executes correctly.
```

After merging this PR I will release it on pypi and then I will remove the code from `edx-platform` and use this.

### CODE REVIEW

**Comment by awais786** [comment]
Internal review in progress.

**Comment by awais786** [comment]
@kdmccormick Please review this PR. Would you please guide us about your testing problem requirement ?

**Comment by kdmccormick** [comment]
@awais786 I can't see the 2u.atlassian.net ticket. Can you copy  the info in, or just link to the original ticket on the openedx.atlassian.net instance?

**Comment by awais786** [comment]
> @awais786 I can't see the 2u.atlassian.net ticket. Can you copy the info in, or just link to the original ticket on the openedx.atlassian.net instance?

https://openedx.atlassian.net/browse/BOM-2585 @kdmccormick

**Comment by awais786** [comment]
https://learning-codejail.sandbox.edx.org/course/course-v1:edX+DemoX+Demo_Course/block-v1:edX+DemoX+Demo_Course+type@sequential+block@641435f80f354c2e83f55162e5d18eec/block-v1:edX+DemoX+Demo_Course+type@vertical+block@4539d85a74f744f9b6f2883740a31f92

new sandbox with `Blank Advanced Problem` + 
```
<problem>
  
<script type="loncapa/python">
import loncapa
x1=random.randint(1,10)
a1=0
  
def answer1(expect, ans):
 a1 = int(ans)
 return (2*x1)==a1

def answer2(expect, ans):
 a2=int(ans)
 return (2*a1)==a2 

</script>  
  
<p>a) What is the double of $x1?</p> 
<customresponse cfn="answer1">
	<textline size="40" label="Integer #1"/>
</customresponse> 

<p>b) What is the double of a)? </p>
<customresponse cfn="answer2">
  <textline size="40" label="Integer #2"/>
</customresponse> 

</problem>
```

**Comment by kdmccormick** [comment]
Thanks Awais, I'll take a look.

File: MANIFEST.in
Comment (kdmccormick) [inline MANIFEST.in]: `codejail-includes` is not a folder within this repo. Perhaps you need to include the sub-packages directly, like:

```suggestion
include eia.py
recursive-include verifiers *.html *.png *.gif *.js *.css *.jpg *.jpeg *.svg *.py
recursive-include loncapa *.html *.png *.gif *.js *.css *.jpg *.jpeg *.svg *.py
```

File: CHANGELOG.rst
Comment (kdmccormick) [inline CHANGELOG.rst]:
```suggestion
* codejail includes moved from edx-platform to individual pypi package.
```

File: openedx.yaml
Comment (kdmccormick) [inline openedx.yaml]: Based on [see OEP-2](https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0002-bp-repo-metadata.html#tags), this would be considered a "library" since it gets run in production.
```suggestion
    - library
```

File: openedx.yaml
Comment (kdmccormick) [inline openedx.yaml]: from https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0010-proc-openedx-releases.html#involving-repos-in-the-open-edx-build-process:
>Libraries that are part of Open edX do not need the key, because they will be pulled in by whatever component uses them as a dependency.
```suggestion
```

File: eia.py
Comment (kdmccormick) [inline eia.py]: Woah, this is janky! We ought to turn these into `assert` statements and move them into a proper unit test file and so they can be run as part of CI.

Can you either do that in this PR, or make a ticket to do it in the future?

**Review by kdmccormick — COMMENTED** [review]
Code is looking pretty good, just a few comments.

A couple concerns with your test code:
* It is bugged: `answer2` will always expect `ans` to be 0, because `ans1` will always be 0 (as the function `answer1` will not modify the global value of `ans1`)
* You import `loncapa`, but do not use it. To truly test these changes, need to ensure that the code from the new `codejail-includes` repository (such as `loncapa.lc_random`) can be imported and executed once `sandbox-packages` is removed from edx-platform. Perhaps it would be easier to verify this in the future, when you have the PR against edx-platfrom to remove `sandbox-packages` and install this new repo?

File: eia.py
Comment (awais786) [reply eia.py]: I did it in this PR.

Comment (awais786) [reply eia.py]: Done in same PR.

File: Makefile
Comment (iamsobanjaved) [inline Makefile]: Fix indentation

File: __init__.py
Comment (iamsobanjaved) [inline __init__.py]: The version isn't being read from here. 

https://github.com/openedx/codejail-includes/pull/1/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R11

**Review by iamsobanjaved — APPROVED** [review]
Few changes otherwise 👍

**Comment by awais786** [comment]
@kdmccormick I am making new sandbox using this [PR](https://github.com/openedx/edx-platform/pull/30402/files)
 I will follow your steps to check the problem. After getting approval I will release it and share platform PR with you.

new sandbox https://learning-awais786.sandbox.edx.org/course/course-v1:edX+DemoX+Demo_Course/block-v1:edX+DemoX+Demo_Course+type@sequential+block@3b59f03fea01465f88f3b4364099b695/block-v1:edX+DemoX+Demo_Course+type@vertical+block@ac1342b877474aa186399fc40650f5fc

**Comment by kdmccormick** [comment]
Sounds good!

**Pull request merged**
